.github/workflows/dev.yml

```diff
@@ -49,9 +49,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        profile:
+          - name: "debug"
+            sterile: ""
```

Comment:
What's the point for the “not-sterile” environment? What are we expecting to catch with this run?

Reply:
The non-sterile environment is there to ensure that the developer's workflow works normally.  I don't want to break normal IDE integrations.  The sterile environment is far too sparse to actually develop anything in.